### PR TITLE
Fix error when concat datatypes have length dividable by 253 == 0

### PR DESCRIPTION
**Pull request by Bairdo**

### CODE REVIEW

**Comment by Bairdo** [comment]
ignore that commit, missed the point about join().

With the test do you want it to be split out so it just tests the concat type, and not the rest of the radius parser? or just rename?

File: chewie/radius_datatypes.py [hunk 1]
```diff
@@ -189,22 +189,22 @@ def parse(cls, packed_value):
         return cls(struct.unpack("!%ds" % len(packed_value), packed_value)[0])
 
     def pack(self, attribute_type):
+
+        def chunks(l):
```
Comment (samrussell) [inline chewie/radius_datatypes.py]: let's use better variable names here - even the word "data" is more meaningful than the letter L

**Comment by samrussell** [comment]
that's looking a lot nicer.

as for the test, think about what the test is for - if we have an edge case where a multiple of 253 is an issue then make that it's own test; equally, we should also test the case with a length that doesn't divide by 253. have a look at https://mtlynch.io/good-developers-bad-tests/#go-crazy-with-long-test-names for some more ideas about this

**Comment by Bairdo** [comment]
Thanks for that.

**Comment by samrussell** [comment]
Looking good, nice work!
